sys-kernel/coreos-sources/files/4.4/0015-Overlayfs-Use-copy-up-security-hooks.patch

```diff
@@ -1,4 +1,4 @@
-From 64ef0efdd90f5aae4fae7c76783b09af53d29dfe Mon Sep 17 00:00:00 2001
+From 78e4150dc214a67ee53b9d2fc0a3d893286d21eb Mon Sep 17 00:00:00 2001
```

Comment:
Note that it doesn't really change, all that's been added is the new context lines.